src/components/Table/Table.stories.jsx

```diff
@@ -0,0 +1,234 @@
+        {Header: 'Last Modified On', accessor: 'lastModifiedOn'}
+    ], []);
+
+    const {
+        getTableProps,
+        getTableBodyProps,
+        headerGroups,
+        rows,
+        prepareRow,
+        selectedFlatRows,
+        state: {selectedRowIds}
+    } = useTable(
+        {
+            data,
+            columns
+        },
+        useRowSelect
+    );
+
+    return (
+        <>
+            <Table {...getTableProps()}>
+                <TableHead>
+                    {headerGroups.map(headerGroup => (
+                        // A key is included in headerGroup.getHeaderGroupProps
+                        // eslint-disable-next-line react/jsx-key
+                        <TableRow {...headerGroup.getHeaderGroupProps()}>
+                            {headerGroup.headers.map(column => (
+                                // A key is included in column.getHeaderProps
+                                // eslint-disable-next-line react/jsx-key
+                                <TableHeadCell {...column.getHeaderProps()}>
+                                    {column.render('Header')}
+                                </TableHeadCell>
+                            ))}
+                        </TableRow>
+                    ))}
+                </TableHead>
+                <TableBody {...getTableBodyProps()}>
+                    {rows.map((row, id) => {
+                        prepareRow(row);
+                        return (
+                            // A key is included in row.getRowProps
+                            // eslint-disable-next-line react/jsx-key
+                            <TableRow
+                                isSelected={row.isSelected}
+                                isHighlighted={id === 1}
+                                {...row.getRowProps()}
+                            >
+                                {row.cells.map(cell => (
+                                    // A key is included in cell.getCellProps
+                                    // eslint-disable-next-line react/jsx-key
+                                    <TableBodyCell {...cell.getCellProps()}>
+                                        {cell.render('Cell')}
+                                    </TableBodyCell>
+                                ))}
+                            </TableRow>
+                        );
+                    })}
+                </TableBody>
+            </Table>
+
+            <section>
+                <pre>
+                    <code className={storyStyles.storyCode}>
+                        {JSON.stringify(
+                            {
+                                selectedRowIds,
+                                'selectedFlatRows[].original': selectedFlatRows.map(d => d.original)
+                            },
+                            null,
+                            2
+                        )}
+                    </code>
+                </pre>
+            </section>
```

Comment:
It will be good if we can add these kinds of information in storybook plugins like `storybook-actions` to display it in a dedicated tab instead of the main canvas.